layouts/shortcodes/common-build-commands.md

```diff
@@ -43,6 +43,15 @@ to avoid the caching, so that you can just Shift+Reload the pages to see the
 new content.
 {{ end }}
 
+{{ if eq $section "clone-lo" }}
+
+```bash
+git clone https://gerrit.libreoffice.org/core libreoffice
+cd libreoffice
+git checkout distro/collabora/{{.Get "lobranch"}}
```

Comment:
Most of the time when we set it lobranch isn't used (clone-online sections), this is probably fine but it could be even better to remove it when we aren't going to be reading it